src/components/GlobalHeader/RightContent.tsx

```diff
@@ -136,14 +136,14 @@ interface UnreadDataType {
 
 function getUnreadData (noticeData) {
  const unreadMsg: UnreadDataType = { notification: 0, message: 0, event: 0 }
- Object.entries(noticeData).forEach(([key, value]) => {
+ for (const [key, value] of Object.entries(noticeData)) {
```

Comment:
shouldn't you use `let` here (instead of `const`)?

Reply:
yes, it is being reassigned. Will update it for all changes here.